README.md

```diff
@@ -149,10 +146,14 @@ Guide to build images again with modification or with different environment vari
 
 Now you'll need to do steps (2) everytime you make a changes and want to build the images again.
 
-**Note:**
+##### Notes:
```

Comment:
Hmm, that's now too small. Can you try `#### Notes:` please?

Reply:
Ok, it will look like this: ![New `notes` variation](https://i.imgur.com/FzSgU4B.png)